en_ewt-ud-dev.conllu

```diff
@@ -279,7 +279,7 @@
 9	against	against	ADP	IN	_	11	case	11:case	_
 10	US	US	PROPN	NNP	Number=Sing	11	compound	11:compound	_
 11	troops	troops	NOUN	NN	Number=Sing	7	obl	7:obl:against	_
-12	when	when	ADV	WRB	PronType=Int	15	mark	15:mark	_
+12	when	when	SCONJ	WRB	PronType=Int	15	mark	15:mark	_
```

Comment:
`PronType=Rel`?

Reply:
Why PronType=Rel? This is an adverbial clause modifying the verb 'arrested'

Reply:
Is interrogative right if it's not in a question construction though? I guess I just don't know how to draw the line between interrogative and relative for adverbial clauses.

Reply:
TBH I think it's neither: it has the same status as 'if' in that it is an SCONJ marking a subordinate adverbial clause. Would you be OK with `_`?

Reply:
@dan-zeman?

Reply:
```suggestion
12	when	when	SCONJ	IN	_	15	mark	15:mark	_
```

I would go without PronType when it is just SCONJ and not PRON. (I know I said that PronType is not restricted to PRON; but I normally use it with PRON, DET and ADV, but not SCONJ. I am not saying that it would be illegal to use it with SCONJ, but since it is difficult to decide, it may be better to omit the feature. If it were ADV instead of SCONJ, I would want to have a PronType in order to signal that it is a pronominal adverb, and then it still might be difficult to decide between Int and Rel, but we can also use `PronType=Int,Rel` in such cases.)

Reply:
Sounds good to me.